python/ucxx/ucxx/_lib_async/tests/test_disconnect.py

```diff
@@ -127,9 +128,9 @@ def test_shutdown_unexpected_closed_peer(caplog, endpoint_error_handling):
         args=(client_queue, server_queue, endpoint_error_handling),
     )
     p2.start()
-    p2.join()
+    p2.join(timeout=30)
     server_queue.put("client is down")
-    p1.join()
+    p1.join(timeout=30)
```

Comment:
**Problem**: The process join operations have a hardcoded timeout of 30 seconds, which may not be sufficient for all scenarios.

**Suggested Change**: Consider using a more dynamic or configurable timeout to accommodate different use cases.


**Severity (1 - 4)**: 2 - MINOR

**Lines**: 131-133

```suggestion
    timeout = 60  # Increase the timeout to 1 minute
    p2.join(timeout=timeout)
    server_queue.put("client is down")
    p1.join(timeout=timeout)
```

-----

<div align="right">
    <small><i>Generated by <b>Llama 3.1 405b</b></i></small>
</div>
<br />


<div align="right">
  <small>Was this helpful?</small>
  <a href="https://codecritic-feedback.nvidia.com?vote=1&comment_guid=ed94d215-525c-4eea-bdf5-67f62dff0449&is_github=true" target="_blank">
    <button>👍</button>
  </a>
  <a href="https://codecritic-feedback.nvidia.com?vote=-1&comment_guid=ed94d215-525c-4eea-bdf5-67f62dff0449&is_github=true" target="_blank">
    <button>👎</button>
  </a>
</div>